ChordProgress/ChordProg.htm

```diff
@@ -56,5 +56,12 @@
       </div>
     </div>
     <div class="test" id="test" style="visibility:hidden"></div>
+    
+    <br>
+    <br>
+    <a href="index.html">
```

Comment:
i think this needs to be a relative link to the folder above it or a full url:
`<a href="../index.html">`